src/memray/reporters/tree.py

```diff
@@ -232,6 +232,7 @@ class TreeApp(App[None]):
         Binding(
             key="e", action="expand_linear_group", description="Expand linear group"
         ),
+        [Binding("ctrl+z", "suspend_process")],
```

Comment:
Extra brackets here as well:
```suggestion
        Binding("ctrl+z", "suspend_process"),
```